sale_order_import_edifact/readme/ROADMAP.rst

```diff
@@ -0,0 +1,5 @@
+..  Enumerate known caveats and future potential improvements.
+    It is mostly intended for end-users, and can also help
+    potential new contributors discovering new features to implement.
+
+* Refactor import by release
```

Comment:
can you elaborate on this?

Reply:
This was an old item that was in the old version, so may I remove it?

Reply:
unless is something useful.... @rmorant ?

Reply:
Not at all, it was just a warning, unclear, that things were changing in the new versions.
can be removed.